src/main/java/games/strategy/engine/lobby/server/ModeratorController.java

```diff
@@ -60,39 +62,27 @@ public boolean isPlayerAdmin(final INode node) {
     return user != null && user.isAdmin();
   }
 
-  @Override
-  public void banMac(final INode node, final Date banExpires) {
-    banMac(node, banExpires != null ? banExpires.toInstant() : null);
+  private Moderator getModeratorForNode(final INode node) {
+    return new Moderator(node.getName(), node.getAddress(), getNodeMacAddress(node));
   }
 
-  private void banMac(final INode node, final Instant banExpires) {
-    assertUserIsAdmin();
-    if (isPlayerAdmin(node)) {
-      throw new IllegalStateException("Can't ban an admin");
-    }
-    final INode modNode = MessageContext.getSender();
-    final String mac = getNodeMacAddress(node);
-    new BannedMacController().addBannedMac(mac, banExpires);
-    final String banUntil = (banExpires == null ? "forever" : banExpires.toString());
-    logger.info(String.format(
-        "User was banned from the lobby(Mac ban). "
-            + "Username: %s IP: %s Mac: %s Mod Username: %s Mod IP: %s Mod Mac: %s Expires: %s",
-        node.getName(), node.getAddress().getHostAddress(), mac, modNode.getName(),
-        modNode.getAddress().getHostAddress(), getNodeMacAddress(modNode), banUntil));
```

Comment:
I verified the code in this overload was identical to that in `banMac(INode, String, Instant)` minus

* the name of the variable used to hold the hashed MAC address (`mac` vs. `hashedMac`), and
* the fact that the first overload calls `getNodeMacAddress()` to obtain the hashed MAC address.